components/InfoSectionPreparation/index.tsx

```diff
@@ -13,40 +13,50 @@ const InfoSectionPreparation = () => {
         Her har vi litt tips for hva det er lurt å tenke på før du reiser
         hjemmefra
       </p>
-      <div className={styles.cardContainer}>
-        {cardData.map(({ title, icon, description }) => (
-          <Card
-            key={title}
-            icon={icon}
-            title={title}
-            description={description}
-          />
-        ))}
-      </div>
+      <CardWrapper>
+        {cardData.map(
+          ({ title, icon, description, buttonHref, buttonText }) => (
+            <Card
+              key={title}
+              icon={icon}
+              title={title}
+              description={description}
+              buttonHref={buttonHref}
+              buttonText={buttonText}
+            />
+          )
+        )}
+      </CardWrapper>
       <p>{/* TODO: Legge til TODO-liste */}</p>
```

Comment:
:eyes:

Reply:
Could remove :p but I'm not on the computer today so I'd rather not do anything about it and fix it later